package.json

```diff
@@ -161,70 +161,83 @@
       "default": "pdf",
       "order": 10
     },
+    "producer": {
+      "description": "Program to use when post-process DVI output in order to produce PDF. This is not used for pdflatex based builders. Currently only supported for latexmk.",
```

Comment:
My comment is not related to the implementation, but I think using the word builder here is a bit confusing. It is already used to indicate latexmk and texify though the latter is going to be deprecated.

Also, XeTeX is not a pdfTeX based engine while LuaTeX is a hybrid successor of pdfTeX and [Omega](https://en.wikipedia.org/wiki/Omega_%28TeX%29). So something like "PDF-producible engines" is a more appropriate terminology IMHO (there might be even better words as I have poor English vocabulary).